### ISSUE TITLE
[BUG] Duplicates with scanned documents through email

### ISSUE THREAD

**Issue by Omaha2002** [paperless-ngx/paperless-ngx#3419]
### Description

We scan and email signed packing slips by customers to paperless through an email rule. 

I've now scanned and emailed a few hundred packing slips manually and Paperless starts to report failed file tasks mentioning there are duplicates. I'm pretty sure there are no duplicates alhough the packing slips might look alike they never are.

![afbeelding](https://github.com/paperless-ngx/paperless-ngx/assets/3179748/44b92bde-cf75-4931-803d-6609a34073d6)

### Steps to reproduce

1. go to scanner, put in signed packing slips
2. wait tíll scanner has send the scanned document as pdf in an email to paperless
3. getting failed task because duplicate of...

### Webserver logs

```bash
None relevant, there are nog webserver errors
```

### Paperless-ngx version

1.14.5

### Host OS

ubuntu 22.04

### Installation method

Docker - official image

### Browser

Firefox, Chroms, Safari

### Configuration changes

None

**Comment by shamoon** [comment]
Can you provide two documents that it says are duplicates that you think are not?

**Comment by stumpylog** [comment]
paperless doesn't use how a document looks for deciding duplication.  It must be a bit for bit identical file.

**Comment by Omaha2002** [comment]
This is one of the errors that shows up in failed tasks:

"escandedocument-05172023-155553.pdf: The following error occurred while consuming Gescandedocument-05172023-155553.pdf: duplicate key value violates unique constraint "documents_document_checksum_75209391_uniq"
DETAIL:  Key (checksum)=(80d0501df5abf1ee3758874ecbb49592) already exists."

Besides the expected error with a duplicate document:

"Gescandedocument-05172023-155553.pdf: Not consuming Gescandedocument-05172023-155553.pdf: It is a duplicate of Gescandedocument-05172023-155553 (#148)"

**Comment by Omaha2002** [comment]
Just to check, I've first scanned a batch with 30 documents as 1 pdf with multiple pages in one email with the scanner to feed it to Paperless. Later on I scanned the same batch with the scanner set to 1 pdf in 1 email, so basically the scanner sending 30 emails with 1 page pdf to Paperless. Will it see the single page pdf's as a duplicate of a page in the 30 page pdf OCR'ed by Paperless?

**Comment by FizzyMUC** [comment]
How come the two documents (where it says it won't consume due to duplication - #148) have the exact same file name? Are you 100% sure it is not the exact same document you are trying to consume again?

**Comment by stumpylog** [comment]
Give the error is not the pre-check duplicate error and the filename looked the same, you're probably;y running the mail fetching too frequently for all of the last fetched PDFs to be processed.  Emails are only marked processed (or deleted, etc) once the attachment is finished consuming.

Increasing the number of workers and increasing the time between fetching will help.
